stdlib/_operator.pyi

```diff
@@ -1,4 +1,5 @@
 # In reality the import is the other way around, but this way we can keep the operator stub in 2and3
```

Comment:
This comment is no longer up to date, as Python 2 and 3 stubs have been separated. We should probably change this to match runtime, but in a separate PR.